Makefile

```diff
@@ -507,6 +507,7 @@ $(1): $(PREFIX)/$(3)/installed/$(1)
 $(PREFIX)/$(3)/installed/$(1): $(PKG_MAKEFILES) \
                           $(PKG_PATCHES) \
                           $(PKG_TESTFILES) \
+                          $($(PKG)_FILE_DEPS) \
```

Comment:
Sure, it should be a separate PR. I would use `$(1)` here to maintain consistency meanwhile.